### PR TITLE
Add SQL basics training

### PR DESCRIPTION
**Pull request by benesch**
I decided this was super important for our internal training. ¯\_(ツ)_/¯  Could definitely use some refinement, but I am clear out of time.

### CODE REVIEW

**Comment by benesch** [comment]
Direct link: http://cockroach-docs-review.s3-website-us-east-1.amazonaws.com/01dd7cef4281208d8253f00ae319db575cadadfe/stable/training/sql-basics.html

**Comment by knz** [comment]
Yeah I like this a lot too.

File: v2.0/training/sql-basics.md [hunk 1]
```diff
@@ -0,0 +1,503 @@
...
+~~~
+
+SQL is even more powerful than we've demonstrated here. Joins and aggregations
+can be combined and nested to express nearly any query you can conceive of. If
+you'd like, [continue learning about CockroachDB
+SQL](../learn-cockroachdb-sql.html) in our docs.
+
+## What's Next?
+
+- [Cluster Startup and Scaling](cluster-startup-and-scaling.html)
```
Comment (jseldess) [inline v2.0/training/sql-basics.md]: As mentioned above, I think it's important to start with basic operations before going into sql. I'd put this no earlier than Data Import.

**Comment by lhirata** [comment]
Closing this PR since I edited / merged on #3556. Thanks!!

**Pull request closed without merging**